File: OOP-js-0/PHP/uberX.php
<?php
require_once('car.php');
class UberX extends Car {
  public $brand;
  public $model;
  protected $passengers;

  public function __construct($license, $driver, $brand, $model) {
    parent::__construct($license, $driver);
    $this->brand = $brand;
    $this->model = $model;
  }


  public function getPassengers() {
    return $this->passengers;
  }

  public function setPassengers($passengers) {
    if ($passengers == 4) {
      $this->passengers = $passengers;
    } else {
      echo "Necesitas 4 pasajeros";
      echo "<br>";
    }
  }
}
?>

File: OOP-js-0/Java/UberX.java
class UberX extends Car {
  String brand;
  String model;

  public UberX(String license, Account driver, String brand, String model) {
    super(license, driver);
    this.brand = brand;
    this.model = model;
  }

  @Override
  void getDataCar() {
    super.getDataCar();
    System.out.println("Brand: " + brand + " Model: " + model);
  }
}
